src/_includes/main-intro/style.css

```
@@ -1,5 +1,181 @@
-.title {
-  font-size: 50px;
+.root {
+  overflow: hidden;
+  position: relative;
+  line-height: normal;
+  background: var(--offwhite);
+  padding: 4.4rem 0;
```

Should this be

```suggestion
  padding: var(--vertical-margin) 0;
```

?

---

Not in this case. The padding at this top of the main module is much larger than other modules.